packages/magento-product/components/ProductPageMeta/ProductPageMeta.tsx

```diff
@@ -3,14 +3,21 @@ import { useProductLink } from '../../hooks/useProductLink'
 import { ProductPageMetaFragment } from './ProductPageMeta.gql'
 
 export function ProductPageMeta(props: ProductPageMetaFragment) {
-  const { name, meta_title, meta_description, url_key, __typename } = props
+  const { sku, categories, price_range, media_gallery, name, meta_title, meta_description, url_key, __typename } = props
   const productLink = useProductLink({ url_key, __typename })
+  const categoryNames = categories?.map((category) => category?.name ?? '')
 
   return (
-    <PageMeta
-      title={meta_title ?? name ?? ''}
-      metaDescription={meta_description ?? name ?? ''}
-      canonical={productLink}
-    />
+      <PageMeta
+          title={meta_title ?? name ?? ''}
+          metaDescription={meta_description ?? name ?? ''}
+          canonical={productLink}
+          sku={sku}
+          name={name}
+          image={media_gallery?.[0]?.url ?? null}
+          categories={categoryNames}
+          price={price_range?.maximum_price?.regular_price?.value ?? 0}
+          salePrice={price_range?.maximum_price?.final_price?.value ?? 0}
+      />
```

Comment:
Might it be an idea to do something like this?

```suggestion
      <>
      <PageMeta
          title={meta_title ?? name ?? ''}
          metaDescription={meta_description ?? name ?? ''}
          canonical={productLink}
      />
      <Head>
              {sku && <>
          <meta property='type' content='product' key='og-type' />
          <meta property='product:retailer_part_no' content={sku} key='og-sku' />
        </>}
        {name && <meta property='og:title' content={name} key='og-title' />}
        {image && <meta property='og:image' content={image} key='og-image' />}
        {price && <meta property='product:price:amount' content={price.toString()} key='og-price' />}
        {salePrice && <meta property='product:sale_price:amount' content={salePrice.toString()} key='og-sale-price' />}
        {categories && categories.map((category, index) => <meta property='product:category' content={category} key={`og-category-${index}`} />)}
      </Head>
      </>
```

Reply:
Good idea, done